go.mod

```diff
@@ -27,7 +27,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.12.1
 	github.com/spf13/cobra v1.3.0
-	github.com/submariner-io/admiral v0.12.1
+	github.com/submariner-io/admiral v0.12.2-0.20220523072532-998de8f36076
```

Comment:
This needs to be updated now that https://github.com/submariner-io/admiral/pull/377 has been merged.